.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .pytest_cache/
 **__pycache__/
 db.sqlite3
+*.db3
```

Comment:
Ensure that excluding `*.db3` files aligns with your project's requirements. If you are using SQLite databases, consider excluding all relevant database file extensions consistently.